packages/devtools/lib/src/timeline/timeline.dart

```diff
@@ -2,28 +2,31 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-import 'dart:async';
-import 'dart:math' as math;
-
 import 'package:meta/meta.dart';
 import 'package:vm_service_lib/vm_service_lib.dart' hide TimelineEvent;
 
 import '../framework/framework.dart';
 import '../globals.dart';
 import '../ui/elements.dart';
 import '../ui/fake_flutter/dart_ui/dart_ui.dart';
+import '../ui/flutter_html_shim.dart';
 import '../ui/icons.dart';
 import '../ui/primer.dart';
 import '../ui/ui_utils.dart';
 import '../vm_service_wrapper.dart';
 import 'frame_flame_chart.dart';
-import 'frame_rendering.dart';
-import 'frame_rendering_chart.dart';
+import 'frames_bar_chart.dart';
 import 'timeline_controller.dart';
 import 'timeline_protocol.dart';
 
-const Color slowFrameColor = Color(0xFFf97c7c);
-const Color normalFrameColor = Color(0xFF4078c0);
+const Color mainCpuColor = Color(0xFFFFCA28);
+const Color mainGpuColor = Color(0xFF9CCC65);
+const Color slowFrameColor = Color(0xFFE50C0C);
+const Color selectedFrameColor = Color(0xFF4078C0);
+final String mainCpuColorCss = colorToCss(mainCpuColor);
```

Comment:
are these string constants added as a performance optimization?

Reply:
it could also make sense to just keep a cache of Color to css strings so colorToCss can be used without worrying it is slow.

Reply:
I added them so we didn't have to import flutter_html_shim in every file that needs one of these color constants. I can add a map here instead of the strings.